src/app/pages/api/openai.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenaiApiType } from '@/app/types/OpenaiApiType';
+
+const fetchData = async (): Promise<OpenaiApiType> => {
+  const response = await fetch('https://api.openai.com/v1/chat/completions', {
```

Comment:
Do you think we should decide on a standard re: fetching from api vs using openai sdk?

Reply:
opted for fetching from API for this ticket to not include the extra dependency but open to challenges on this !

Reply:
Cool, makes sense within next app, can still discuss for our one-shot code